src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -104,12 +104,13 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 					exact,
 					undefined,
 					TriggeredByEnum.PLACES_SEARCH,
-					action
+					action,
+					isNLChecked
 				);
 				setIsSearching(false);
 			}, 200);
 		},
-		[mapRef, setIsSearching, search]
+		[mapRef, setIsSearching, search, isNLChecked]
```

Comment:
`isNLChecked` is terse, which is good.  But it speaks of the checkbox over the feature. Would you be open to use `isNLSearchEnabled`?